src/variables.json

```diff
@@ -75,18 +75,33 @@
 
   "focus-shadow": "0 0 0 3px rgba(137, 199, 216, 0.65)",
 
+  "font-size-h1": "36px",
+  "font-size-h2": "30px",
+  "font-size-h3": "24px",
+  "font-size-h4": "20px",
+  "font-size-h5": "16px",
+
+  "line-height-h1": "45px",
+  "line-height-h2": "36px",
+  "line-height-h3": "30px",
+  "line-height-h4": "25px",
+  "line-height-h5": "20px",
+
   "font-size-xl": "30px",
-  "font-size-l": "18px",
-  "font-size-m": "15px",
+  "font-size-l": "20px",
+  "font-size-m": "16px",
+  "font-size-ms": "14px",
   "font-size-s": "12px",
   "font-size-xs": "10px",
 
   "line-height-xl": "45px",
   "line-height-l": "30px",
   "line-height-m": "24px",
```

Comment:
@samanpwbb Since `font-size-m` increased by a pixel, what do you think about increasing `line-height-m` to `25px`? I tested these changes in a docs site and noticed the line-height felt a little tight. 


`line-height: 24px` - current | `line-height: 25px` - proposed
:---|:---
<img width="1002" alt="Screen Shot 2021-04-01 at 3 31 05 PM" src="https://user-images.githubusercontent.com/2180540/113344865-a8b9d980-92ff-11eb-9b0f-bafaab0006b6.png"> | <img width="1002" alt="Screen Shot 2021-04-01 at 3 31 23 PM" src="https://user-images.githubusercontent.com/2180540/113344873-aa839d00-92ff-11eb-8a69-8d273eb24146.png">

Reply:
16/24 is a 1.5 ratio, which is the recommended rule of thumb for body text. 15/24 is a 1.6 ratio, and I agree in some contexts it is more elegant. 

I am hesitant to raise the default line height to 25, since 25 is misaligned with Assembly's 6px baseline grid. As a result, certain layouts would be off by a few pixels unless heights on container elements get hard-coded.

Here are some reasonable ways forward:

1. Revert to 15/24 for txt-m
2. Use larger, more elegant line heights for text inside `prose`.
3. Add dedicated line height classes?
4. Stick with 16/24

Reply:
That makes sense. Let's stick with what we have here, I'd much prefer `txt-m` be at 16px and stay aligned.